src/Typeahead.react.js

```diff
@@ -158,7 +180,10 @@ Typeahead.propTypes = {
 
 Typeahead.defaultProps = {
   renderMenu: (results, menuProps) => (
-    <TypeaheadMenu {...menuProps} options={results} />
+    <TypeaheadMenu
```

Comment:
Doesn't look like anything changed here? I'd omit formatting changes from your PR, as it just adds noise.